ytypes/container.go

```diff
@@ -106,12 +106,13 @@ func validateContainer(schema *yang.Entry, value ygot.GoStruct) util.Errors {
 }
 
 // unmarshalContainer unmarshals a JSON tree into a struct.
-//   schema is the schema of the schema node corresponding to the struct being
-//     unmarshaled into.
-//   parent is the parent struct, which must be a struct ptr.
-//   jsonTree is a JSON data tree which must be a map[string]interface{}.
-//   opts is the set of options that should be used when unmarshalling the JSON
-//     into the supplied parent.
+//
+//	schema is the schema of the schema node corresponding to the struct being
+//	  unmarshaled into.
+//	parent is the parent struct, which must be a struct ptr.
+//	jsonTree is a JSON data tree which must be a map[string]interface{}.
+//	opts is the set of options that should be used when unmarshalling the JSON
+//	  into the supplied parent.
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.